lib/gcloud/datastore/dataset.rb

```diff
@@ -239,19 +242,31 @@ def delete *entities_or_keys
       #
       # +query+::
       #   The Query object with the search criteria. (+Query+)
+      # +options+::
+      #   An optional Hash for controlling additional behavior. (+Hash+)
+      # <code>options[:namespace]</code>::
+      #   The namespace the query is to run within. (+String+)
       #
       # === Returns
       #
       # Gcloud::Datastore::Dataset::QueryResults
       #
-      # === Example
+      # === Examples
       #
       #   query = Gcloud::Datastore::Query.new.kind("Task").
       #     where("completed", "=", true)
       #   tasks = dataset.run query
       #
-      def run query, opts = {}
-        response = connection.run_query query.to_proto, opts
+      # The query can optionally run within namespace when the +namespace+
+      # option is provided:
```

Comment:
Maybe: "The query can be scoped to a namespace with the +namespace+ option." ?
And link namespace to external doc if there's a good explanation of it somewhere?